packages/x-codemod/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-codemod",
-  "version": "6.0.0-alpha.11",
+  "version": "6.0.0-alpha.12",
```

I just thought that we should consider again switching to a fixed version in `lerna.json` instead of `independent` to avoid these issues? 🤔 
It made sense in master, having different pickers and grid version, but on next it is currently aligned 🤔 
```suggestion
  "version": "6.0.0-alpha.13",
```